DIR_OUTLINE.md

```diff
@@ -0,0 +1,35 @@
+
+**backend**
+  - endpoints: Endpoints are functions used to send information to the frontend, in other words they handle HTTP requests. This directory contains functions that the endpoint functions for post and get. The get endpoints is where you send get requests to, and the post endpoint is where you send the post requests to. 
+  - globals: Contains the configurations for firebase
+  - helpers:This contains all of the functions used in the backend, for example add_donation, to help the endpoint add a donation
+  - types: contains the structs (similar to classes) of donation and user-data
+  - .env.template the configuration for starting the firebase project
+  - .gitignore contains files for github to ignore such as sensitive information like our project key
+  - Dockerfile: script to assemble an image
+  - compose.yml This is a Docker Compose file. It's used to define and run multi-container Docker applications. It uses YAML syntax to describe the services, networks, and volumes for a complete application stack.
+  - firestore.go contains functions and interfaces for interacting with firestore
+  - go.mod This is the Go module file. It's automatically generated by the Go tooling and it describes the module's dependencies. It allows for versioned dependencies and reproducible builds.
+  - go.sum This is the Go checksum file. It's also automatically generated by the Go tooling. It contains the expected cryptographic checksums of the content of specific module versions. It's used to ensure that these modules have not been tampered with.
+  - main.go Initilizes the log objects, and the routes
+  - main_test.go Contains testing functions for the backend functions to make sure they work as expected 
+**frontend**
+  - tests test for the frontend
+  - components containsn the components in used, for example the DonationCard
+  - cypress: This directory contains end-to-end tests written with Cypress.
+  - lib: This directory contains library code / utility functions
+  - media: Contains media files like images or videos
+  - pages contains the pages of our website, for example the donationList page, or the home page
+  - public: static files that can be accessed directly by the browser.
+  - styles contains the styling for our website
```

Comment:
colon, mention CSS, as well as the fact that we use inline tailwind CSS styles hence why there aren't very many css files